sql-select/actor-forty.sql

```diff
@@ -0,0 +1,4 @@
+select "firstName",
+  "lastName"
+  from "actors"
+  where "actorId" = '40';
```

Comment:
Generally, if a value is a non-string-type, you shouldn't need to wrap it in single-quotes.

Reply:
Would numbers be the only non string type in sql databases or are there other data types that can be stored here?